gammapy/datasets/fermi.py

```diff
@@ -216,6 +218,135 @@ def fetch_fermi_diffuse_background_model(filename='gll_iem_v02.fit'):
 
     return filename
 
+class Fermi3FGLObject(object):
+    """TODO: Doccomments
+    """
+    fermi_cat = fetch_fermi_catalog('3FGL')
+
+    x_bins = np.log10([0.03, 0.1, 0.3, 1, 3, 10, 100])
+
+    y_labels = ['Flux30_100', 'Flux100_300', 'Flux300_1000',
+                'Flux1000_3000', 'Flux3000_10000', 'Flux10000_100000']
+
+    def __init__(self, source_name):
+        self.name_3FGL = source_name
```

Comment:
I'm not sure if it is beneficial to add an attribute for every catalog column to the `Fermi3FGLObject`, that basically duplicates the column name. It could be better just to inherit from `astropy.table` or `astropy.Row` and then access data by e.g. `self['RAJ2000']`. Have you already discussed with @cdeil about a possible API?